cellengine/resources/experiment.py

```diff
@@ -334,13 +334,13 @@ def delete_gate(
         return self.client.delete_gate(self._id, _id, gid, exclude)
```

Comment:
`src(): run black and fix mkdocs errors` - also need to run this on each commit individually so no commits in the main branch have known issues.

Reply:
:+1: